api/src/main/java/org/openmrs/module/idgen/SequentialIdentifierGenerator.java

```diff
@@ -173,4 +173,19 @@ public String getBaseCharacterSet() {
 	public void setBaseCharacterSet(String baseCharacterSet) {
 		this.baseCharacterSet = baseCharacterSet;
 	}
+	
+	/**
+	 * @return the nextSequenceValue
```

Comment:
This file does not look like it will compile.

Reply:
That's the way it was in the reverted commit.